ci/test.sh

```diff
@@ -125,13 +125,13 @@ CONFIG_KUNIT=y
 CONFIG_RUST_KERNEL_DOCTESTS=y
 EOF
 
-  make -C linux KBUILD_VERBOSE=1 LLVM=1 -j$(($(nproc) + 1)) \
+  make -C linux LLVM=1 -j$(($(nproc) + 1)) \
```

Comment:
I think this might still be useful but either way. Yeah maybe it's too verbose.

Reply:
Yeah, I wasn't sure if you intended to keep it or it was just for debugging the issue, so I left the tree as it was before this happened.

It is fairly verbose, yeah; on the other hand, we build a tiny part of the kernel, so it should be OK.